packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcher.tsx

```diff
@@ -26,7 +26,7 @@ const _OrganizationSwitcher = withFloatingTree(() => {
           ref={reference}
           onClick={toggle}
           isOpen={isOpen}
-          aria-controls={switcherButtonMenuId}
+          aria-controls={isOpen ? switcherButtonMenuId : undefined}
```

Comment:
@anagstef are you sure ? maybe you simple wanna add `aria-expanded` ?

Reply:
@panteliselef the `aria-controls` attribute needs to point to an element in HTML that exists in the DOM. The UserButton popover gets added to the DOM only when you click on the button.